src/index.js

```diff
@@ -2,15 +2,13 @@ import React, { Component } from 'react';
 import propTypes from 'prop-types';
 import keyframe from 'keyframe';
 import raf from 'raf';
-import {
-  css,
-  easing,
-  keyframes,
-  spring,
-  tween,
-  value,
-  transform,
-} from 'popmotion';
+import css from 'stylefire/css';
+import easing from 'popmotion/easing';
```

Comment:
You got here the same error that you have in Popmotion's documentation.
It's either `import * as easing from 'popmotion/easing'; ` or in this case `import { linear } from 'popmotion/easing';`